pkg/windows/windows.go

```diff
@@ -1055,3 +1081,10 @@ func rmDirCmd(dirName string) string {
 func getHNSNetworkCmd(networkName string) string {
 	return "Get-HnsNetwork | where { $_.Name -eq '" + networkName + "'}"
 }
+
+// SplitPath splits a Windows file path into the directory and base file name.
+// Example: 'C:\\k\\bootstrap-kubeconfig' --> dir: 'C:\\k\\', fileName: 'bootstrap-kubeconfig'
+func SplitPath(filepath string) (dir string, fileName string) {
```

Comment:
Looks like this logic is duplicated in [splitWindowsPath](https://github.com/openshift/windows-machine-config-operator/pull/1279/commits/895a426f5807419fc181bf04d7996a22ea185166#diff-d5ff1964cc140f787c316cc3b809cd4bf63566860d34d4716fb24f1b171d30b7R701). Could this be modularized?

Reply:
Yep, splitWindowsPath was supposed to be removed in favor of SplitPath (bad rebase). I'll fix next time I push.